lesson02/assignment/css/_business.card.scss

```diff
@@ -0,0 +1,50 @@
+//Sets font for whole card
+
+main {
+  font-family: Verdana;
+  //Variable for gray used by all fonts on page
+  color: $standard-gray;
+  //Uses nesting to make "category" text (employee's name and categories such as company) bold
+  .category {
+    font-weight: bold;
+  }
+  //More nesting to make font on left side of card larger
+
+  .personal-info {
+    font-size: 130%; //Nesting to Italicize title on left side of card
+
+    .title {
+      font-style: italic;
+    }
+  }
+  //Seting font size for right side of card
+
+  .professional-info {
+    font-size: 115%;
+  }
+}
+//Creates general, overall field for information
+
+.card {
+  height: 450px;
+  width: 850px;
```

Comment:
Same with the height, you'd prefer an `auto` height and utilize more padding to create the design.